receiver/journaldreceiver/README.md

```diff
@@ -0,0 +1,26 @@
+## `Journald Receiver`
+
+Parses Journald events using using the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
+
+## Configuration
+
+| Field                  | Default          | Description                                                                                                        |
+| ---                    | ---              | ---                                                                                                                |
+| `directory`            | /run/log/journal | A directory containing journal files to read entries from     |
```

Comment:
The underlying operator will by default read from either `/run/log/journal` or `/run/journal`, whichever is present. Can you please note both here?

[journald_input docs](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/operators/journald_input.md)